dev/io.openliberty.org.jboss.resteasy.common/resources/META-INF/services/org.eclipse.microprofile.config.spi.ConfigProviderResolver

```diff
@@ -0,0 +1,2 @@
+# Intended only for Java SE environments - should NOT be exposed by the ResourceProvider in the bnd file
```

Comment:
do we need to explicitly NOT export this package in bnd?

Reply:
The package is fine to export.  We just not explicitly NOT add this file to the `app_resources` list of META-INF files that we "publicize" via the ResourceProvider service - i.e.
```
app-resources= \
  META-INF/services/javax.ws.rs.client.ClientBuilder | \
  META-INF/services/javax.ws.rs.sse.SseEventSource\$Builder | \
  META-INF/services/javax.ws.rs.sse.SseEventSource.Builder | \
  META-INF/services/javax.ws.rs.ext.RuntimeDelegate | \
  META-INF/services/javax.ws.rs.ext.Providers
```

If we do, then things might go awry when users try to use the client inside Liberty servers.